Hide-Channels/HideChannels.plugin.js

```diff
@@ -329,7 +329,7 @@ module.exports = class HideChannels {
 }
 
 /* Attached CSS to sidebar */
-.${config.constants.hideElementsName} {
+html:not(.visual-refresh) .${config.constants.hideElementsName}.${config.constants.hideElementsName} {
```

Comment:
First off thanks for making a pull request. I appreciate the help.

The above evaluates to the following:
`html:not(.visual-refresh) .hideChannelElement.hideChannelElement {`

Yet in your post #157 you mention the following:
`html:not(.visual-refresh) .sidebar_a4d4d9:not(.hidden_a4d4d9) {`

I'm not a CSS wizard so I'm having trouble figuring out if what you added is what you meant to add or not.

I'd love to hear back from you. Thanks in advance.

Reply:
Ah! I think I see what you meant:
![image](https://github.com/user-attachments/assets/a12064d9-7294-49b7-9b76-e83a95942692)

Reply:
Ah, interesting: https://stackoverflow.com/a/57817383

So basically `!important` on steroids.